docs/source/futures.rst

```diff
@@ -872,6 +872,12 @@ All operations that require talking to the remote worker are awaitable:
 
        n = await counter.n  # attribute access also must be awaited
 
+Generally, all I/O operations that trigger computations (e.g. ``to_parquet``) should be done using the ``compute=False`` 
+parameter to avoid asynchronous blocking:
+
+.. code-block:: python
+
+   df3 = await client.compute(dd.to_parquet('/tmp/some.parquet', compute=False))
```

Comment:
Apologies for the delayed review. A couple of comments:

- I think the result of this computation is `None`. Assigning it a variable name of `df3`, while technically valid, is a bit confusing as it makes me think it's a `DataFrame`
- The `dd.to_parquet(...)` signature needs a `DataFrame` passed as the first argument

Reply:
I pushed the green button too fast. I'll fix this up.